src/app/Modal/DeleteWarningUtils.tsx

```diff
@@ -25,6 +25,7 @@ export enum DeleteOrDisableWarningType {
   DeleteCustomTargets = 'DeleteCustomTargets',
   DeleteDashboardLayout = 'DeleteDashboardLayout',
   DeleteLayoutTemplate = 'DeleteLayoutTemplate',
+  ClearAllDashboardCards = 'ClearAllDashboardCards',
```

Comment:
We should rename this enum member to `ClearDashboardLayout` to match the display text and above member `DeleteDashboardLayout`.

Reply:
```ts

export enum DeleteOrDisableWarningType {
...output_omitted...
ClearDashboardLayout = 'ClearDashboardLayout'
}


export const ClearDashboardLayout: DeleteOrDisableWarning = {
  id: DeleteOrDisableWarningType.ClearDashboardLayout,
  title: 'Clear Dashboard Layout,
  label: 'Clear Dashboard Layout',
  description: 'If you click clear, all cards in the current dashboard layout will be removed.',
  ariaLabel: 'Clear dashboard layout warning',
};
```

Reply:
That included the suggested from Andrew above: https://github.com/cryostatio/cryostat-web/pull/1078#discussion_r1296118842

Reply:
@aali309 :))